extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -373,7 +382,9 @@ export abstract class AzureAuth implements vscode.Disposable {
 			throw new AzureAuthError(msg, 'Adding account to cache failed', undefined);
 		}
 		try {
+			Logger.pii(`Saving access token`, [{ name: 'access', objOrArray: accessToken }], []);
 			await this.tokenCache.saveCredential(`${accountKey.accountId}_access_${resource.id}_${tenant.id}`, JSON.stringify(accessToken));
+			Logger.pii(`Saving refresh token`, [{ name: 'refresh', objOrArray: refreshToken }], []);
```

Comment:
Why the array with a single object? Seems easier just to directly embed the strings in the original string